WootricSDK/WootricSDK/WTRiPADSurveyViewController.m

```diff
@@ -37,6 +37,7 @@
 #import "Wootric.h"
 #import "WTRSurveyDelegate.h"
 #import <Social/Social.h>
+#import <WootricSDK/WootricSDK-Swift.h>
```

Comment:
Same question as above

Reply:
When we are using swift code in Objective C then we have to import swift header file like above.